Earthfile

```diff
@@ -1,28 +1,36 @@
 VERSION 0.7
+
 ARG --global go_version=1.20.2
 ARG --global golangci_lint_version=1.52.2
 ARG --global bufbuild_version=1.15.1
-FROM golang:$go_version-alpine
+ARG --global migrate_version=4.15.2
+ARG --global python_version=3.11 # for sqlfluff
+ARG --global sqlfluff_version=2.0.2
```

Comment:
I prefer keeping all versions at the top of the Earthfile because it makes them easy to change and keeps them organized in one place.